fat-jar/fat-jar-pom.xml.in

```diff
@@ -0,0 +1,677 @@
+<!-- Generated by Makefile on Tue Mar 3 17:23:42 EST 2015 -->
```

Comment:
I was going to do that, but the FatJar is **really** fat so I wanted maven to kick out two seperate artifacts, the original "thin" Jar and the FatJar. I suggest both should be generated and both should have different maven coordinates.